vortex-scalar/src/binary.rs

```diff
@@ -89,15 +89,15 @@ impl TryFrom<Scalar> for Option<Buffer> {
     }
 }
 
-impl From<&[u8]> for Scalar {
-    fn from(value: &[u8]) -> Self {
-        Scalar::from(Buffer::from(value))
+impl From<&'static [u8]> for Scalar {
+    fn from(value: &'static [u8]) -> Self {
+        Scalar::from(Buffer::from_owner(value))
```

Comment:
this seems sus (passing a ref to from_owner)

Reply:
It's static, it's all good